cle/backends/relocations/pearm.py

```diff
@@ -0,0 +1,27 @@
+import logging
+from . import generic_pe
+
+l = logging.getLogger('cle.relocations.pearm')
+
+arch = 'pearm'
```

Comment:
Passed to [get_relocations](https://github.com/sraboy/cle/blob/3d17c5d6b648a4b14df66f1df505c9b3c37881a5/cle/backends/relocations/__init__.py#L40) by [elf backend](https://github.com/sraboy/cle/blob/3d17c5d6b648a4b14df66f1df505c9b3c37881a5/cle/backends/elf/elf.py#L548), so I was just matching it. It does feel a bit wonky since the module's name is the arch.